README.md

```diff
@@ -1 +1,36 @@
-# Math-Magicians
\ No newline at end of file
+# Math Magicians
+
+>  In this project I'm making a Single Page App (SPA) to make simple calculations and read a random math-related quote.
+
```

Comment:
- [ ] Kindly add instructions on how to clone your repository and get it working on the terminal.
- [ ] Also add all the instructions and commands needed to install all the dependencies and start your app development mode. Commands like, `npm install`, `npm start`.